addons/payment_adyen/static/src/js/payment_form.js

```diff
@@ -173,6 +173,35 @@ odoo.define('payment_adyen.payment_form', require => {
                             showPaymentMethods: true,
                             showPayButton: false,
                             setStatusAutomatically: true,
+                            onSelect: (component) => {
```

Comment:
No need for the parenthesis.
```suggestion
                            onSelect: component => {
```